src/Microsoft.OpenApi.Hidi/Program.cs

```diff
@@ -14,26 +15,61 @@ static async Task<int> Main(string[] args)
+            resolveExternalOption.AddAlias("-ex");
+
+            var filterByOperationIdsOption = new Option("--filter-by-operationids", "Filters OpenApiDocument by OperationId(s) provided", typeof(string));
+            filterByOperationIdsOption.AddAlias("-op");
+
+            var filterByTagsOption = new Option("--filter-by-tags", "Filters OpenApiDocument by Tag(s) provided", typeof(string));
+            filterByTagsOption.AddAlias("-t");
+
+            var filterByCollectionOption = new Option("--filter-by-collection", "Filters OpenApiDocument by Postman collection provided", typeof(string));
+            filterByCollectionOption.AddAlias("-c");
 
             var validateCommand = new Command("validate")
             {
-                new Option("--input", "Input OpenAPI description file path or URL", typeof(string) )
+                descriptionOption,
+                logLevelOption
             };
-            validateCommand.Handler = CommandHandler.Create<string>(OpenApiService.ValidateOpenApiDocument);
+
+            validateCommand.Handler = CommandHandler.Create<string, LogLevel>(OpenApiService.ValidateOpenApiDocument);
 
             var transformCommand = new Command("transform")
             {
-                new Option("--input", "Input OpenAPI description file path or URL", typeof(string) ),
-                new Option("--output","Output OpenAPI description file", typeof(FileInfo), arity: ArgumentArity.ZeroOrOne),
-                new Option("--version", "OpenAPI specification version", typeof(OpenApiSpecVersion)),
-                new Option("--format", "File format",typeof(OpenApiFormat) ),
-                new Option("--inline", "Inline $ref instances", typeof(bool) ),
-                new Option("--resolveExternal","Resolve external $refs", typeof(bool)),
-                new Option("--filterByOperationIds", "Filters OpenApiDocument by OperationId(s) provided", typeof(string)),
-                new Option("--filterByTags", "Filters OpenApiDocument by Tag(s) provided", typeof(string)),
-                new Option("--filterByCollection", "Filters OpenApiDocument by Postman collection provided", typeof(string))
+                descriptionOption,
+                outputOption,
+                versionOption,
+                formatOption,
+                logLevelOption,
+                inlineOption,
+                resolveExternalOption,
+                filterByOperationIdsOption,
+                filterByTagsOption,
+                filterByCollectionOption
             };
-            transformCommand.Handler = CommandHandler.Create<string, FileInfo, OpenApiSpecVersion?, OpenApiFormat?, string, string, string, bool, bool>(
+
+            transformCommand.Handler = CommandHandler.Create<string, FileInfo, OpenApiSpecVersion?, OpenApiFormat?, LogLevel, string, string, string, bool, bool>(
                 OpenApiService.ProcessOpenApiDocument);
```

Comment:
```suggestion
                await OpenApiService.ProcessOpenApiDocument);
```

Reply:
ah ignore my confused self, but please double check the system.commandline API recognizes it's now a tasks and blocks on it, I ran into that issue with kiota a while ago and this leads to very confusing behaviours

Reply:
Yep sorry, as long as the command invoke blocks on the task we're good